.env.local

```diff
@@ -23,3 +23,4 @@ MENTOR_WEBHOOK_ID=
 ZIPCODE_GETTER_USERID=
 JOBSEARCH_API_ID=
 JOBSEARCH_API_KEY=
+NEXT_PUBLIC_GOOGLE_ANALYTICS=YOUR_GOOGLE_ANALYTICS_ID=UA-73856435-2
```

Comment:
Is this correct?

Reply:
Our env.local shout be in our gitignore. I will check it out to make sure its fixed.

Reply:
Should be fixed now